server/src/com/scraniel/auctioneero/AuctionResponse.java

```diff
@@ -1,14 +1,18 @@
-package auctioneero;
+package com.scraniel.auctioneero;
+
+import java.util.UUID;
 
 public class AuctionResponse 
 {
 	private boolean isActionSuccess;
 	private String outputMessage;
+	private UUID id;
 	
-	public AuctionResponse(boolean isActionSuccess, String outputMessage)
+	public AuctionResponse(boolean isActionSuccess, String outputMessage, UUID id)
 	{
 		this.isActionSuccess = isActionSuccess;
```

Comment:
once you have a different naming convention for private member variable, remove the "this." :)